### ISSUE TITLE
Individual steps have dissapeared from CLI watch

### ISSUE THREAD

**Issue by simster7** [argoproj/argo-workflows#2088]
When running a workflow with `argo submit --watch` the individual workflow steps are no longer visible. UI is unaffected.

**Comment by simster7** [comment]
Offending commit is #1974 
```
3293c83f6170ad4dc022067bb37f12d07d2834c1 is the first bad commit
commit 3293c83f6170ad4dc022067bb37f12d07d2834c1
Author: Alex Collins <alexec@users.noreply.github.com>
Date:   Tue Jan 21 17:18:38 2020 -0800

    feat: Add version to offload nodes. Fixes #1944 and #1946 (#1974)

```

**Comment by simster7** [comment]
Not actually a regression, but this behavior appears when `ALWAYS_OFFLOAD_NODE_STATUS` is set to `true` _and_ the CLI is not aimed at the Argo Server (by setting `ARGO_SERVER` and `ARGO_TOKEN`) accordingly.

@alexec To provide better documentation and display a warning when the CLI detects this behavior.

**Comment by alexec** [comment]
Will be fixed in #2048
